tools/test-tools/bin/assign-test-ports

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env node
 const mod = require("../dist/assignTestPorts.js");
+const nconf = require("nconf");
```

Comment:
I was going to keep test-tools dependency-less, but figured it's not much overhead to add nconf and we use it elsewhere in our test infra (e2e test scripts). Open to writing some minimal parsing logic ourselves if reviewer would prefer that though; we don't *need* a particularly reusable or ergonomic thing here.

Reply:
When do dependencies for assign-test-ports get installed? Is there an existing use case where the dependencies are not installed?
I think using helper is good, but not confident this won't breaking existing workflows. :/

Reply:
All of our usage of test-tools should just come via normal dep declaration in package.json, so transitive dependencies should be fine. e.g. see changes in https://github.com/microsoft/FluidFramework/pull/23285/files#diff-483384132d69f74744804315cbd004d361f840a7a007d9cfaca2714c69428493

Reply:
+1, I think this won't be an issue. A bit related, I have [another PR open](https://github.com/microsoft/FluidFramework/pull/23211) to make test-tools a `@fluid-private` package in the client release group, so all its uses would be limited to it being a dev dependency during the build.

Reply:
So, if `test-tools` is becoming private, then I don't think this makes sense to merge to main. Something needs to support the LTS branch. Can either the privatization also happen in LTS OR is there a test-tools release and development branch that this should accumulate to?

Reply:
Fair. I'd propose test-tools 2.x as the one that keeps supporting LTS, so we could merge this to release/test-tools/2.0 instead of main, release 2.1.0, and have that be the (hopefully) final piece that fixes the LTS build. I don't expect that we'll need many changes to test-tools going forward for the sake of LTS, so I'd be ok doing all of it as patches/minors on top of that release branch.

Reply:
I was implicitly thinking of adding this change to the test-tools 2.x release branch anyway in order to consume it from LTS, just hadn't thought that it indeed doesn't make much sense in main.